backend/internal/adapters/secondary/repositories/user.go

```diff
@@ -193,3 +193,29 @@ func (r *userRepository) DeclineFriendRequest(requesterID int64, receiverID int6
 
 	return nil
 }
+
+func (r *userRepository) UpdateAvatar(id int64, newAvatar []byte) error {
+	sql, _, _ := qb.Update("users").Where(goqu.C("id").Eq(id)).Set(goqu.Record{
```

Comment:
i thnk goqu has issues with bytes. You may need to use prepared statements instead. Can take a look at the gem repository. I did something similar

Reply:
updated, tq!